### ISSUE TITLE
Deal with raw HTML snippets post pandoc conversion

### ISSUE THREAD

**Issue by peterjc** [OBF/wiki#2]
There are some HTML snippets left in the post conversion markdown. These need to be reviewed and addressed individually.

~~e.g. the main page used HTML for the blue donation box layout, with mediawiki text within it. This resulted in Markdown within an HTML block which does not render as expected. This might be best resolved by turning the contents of the blue box into HTML too?~~ Done in 994f42187cd481c2424b9909eca9988fb23e34c1

**Comment by peterjc** [comment]
e.g. Logos on https://github.com/OBF/wiki/edit/main/BOSC_2014.md

**Comment by peterjc** [comment]
With the progress on #10 the logo tables were looking much better, they just needed their borders turned off. I've not spotted any other raw HTML issues.

These attempts failed - note we do not want to the actual tables in the BOSC schedule pages:

```
$ sed -i 's#<table border="0">#<table style="border: none !important;">#g' BOSC_????.md BOSC_????_Panel.md Codefest_*.md
```

```
$ sed -i 's#<table>#<table border="0">#g' BOSC_????.md BOSC_????_Panel.md Codefest_*.md
```

**Comment by peterjc** [comment]
Looks like we'd need to override this at the <td> level, or tweak the theme - quoting https://github.com/pages-themes/cayman/blob/master/_sass/jekyll-theme-cayman.scss

```css
  table {
    display: block;
    width: 100%;
    overflow: auto;
    word-break: normal;
    word-break: keep-all; // For Firefox to horizontally scroll wider tables.
    -webkit-overflow-scrolling: touch;

    th {
      font-weight: bold;
    }

    th,
    td {
      padding: 0.5rem 1rem;
      border: 1px solid $table-border-color;
    }
  }
```

e.g. Set ``$table-border-color`` to the page background?

**Comment by peterjc** [comment]
Transparent worked, although it might now be worth forcing a border on the schedule pages, e.g. https://obf.github.io/wiki/BOSC_2017_Schedule

**Comment by peterjc** [comment]
Missing columns on the BOSC 2017 schedule fixed in 2c7ee0419a0e1bc3a8ecbefa6858d06b57ac90e6

Another table issue had dropped the Bina logo from some sponsorship listings, fixed in 55c131e26fa13ab654fb7c14e3926cf97c4c17c4 and 5dc26449985ee1c890d9179e520aee090f7df386

I can't say for sure that I have checked every page by eye, but I'm not aware of anything further. Which is not to say there are bits where the layout could be improved ;)
